python/kserve/kserve/storage/test/test_storage.py

```diff
@@ -144,8 +144,10 @@ def test(_):
 def test_mock_gcs(mock_storage):
     gcs_path = 'gs://foo/bar'
     mock_obj = mock.MagicMock()
-    mock_obj.name = 'mock.object'
-    mock_storage.Client().bucket().list_blobs().__iter__.return_value = [mock_obj]
+    mock_obj.name = 'bar/'
+    mock_obj1 = mock.MagicMock()
+    mock_obj1.name = 'bar/mock.object'
+    mock_storage.Client().bucket().list_blobs().__iter__.return_value = [mock_obj, mock_obj1]
```

Comment:
Can we add more tests for GCS like we have for s3/azure ? not sure what we are asserting here

Reply:
IIRC, I saw something related to multimodel, a feature request I guess, wouldn't be better to align with it first?